File: kodovi/PosiljkaWebApi_rad/Posiljka.Api/ViewModels/KorisnikPregledVM.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Posiljka.Api.ViewModels
{
    public class KorisnikPregledVM
    {
        public class Row
        {
            public int? id;
            public string ime;
            public string prezime;
            public string opstina;
        }

        public List<Row> rows;
    }
}

File: PosiljkaApp_video6a/app/src/main/java/android/fit/ba/posiljka/data/KorisnikPregledVM.java
package android.fit.ba.posiljka.data;

import java.util.List;

public class KorisnikPregledVM
    {
        public static class Row
        {
            public Integer id;
            public String ime;
            public String prezime;
            public String opstina;
            public String drzava;
        }
        public List<Row> rows;
    }
